### PR TITLE
Fix flaky test for Windows CI

### PR DESCRIPTION
**Pull request by ybiquitous**
Only the first case fails. So I guess just retrying the failed case is a simpler solution.

See:
- https://github.com/ruby/vscode-typeprof/actions/runs/5138725523/jobs/9248353039
- https://mochajs.org/#retry-tests
- #112

### CODE REVIEW

**Comment by ono-max** [comment]
Is there an error similar to https://github.com/golang/vscode-go/blob/master/test/gopls/extension.test.ts#L229-L253?

**Comment by ybiquitous** [comment]
I'm not 100% sure, but it seems the same. 👀

**Comment by ybiquitous** [comment]
Surely, it seems we could also use [`sleep()`](https://github.com/golang/vscode-go/blob/9ae57e8f912109376520847777804a6346ac39e9/test/gopls/extension.test.ts#L146-L148) or implement a custom retrying like vscode-go.

**Comment by ono-max** [comment]
I prefer the approach of vscode-go to the current approach since it is a more accurate test. What do you think?

**Comment by ybiquitous** [comment]
Okay, let me try the preferable approach. 💪🏼

File: src/test/suite/extension.test.ts [hunk 1]
```diff
@@ -17,19 +17,31 @@ suite('completion', () => {
...
-        )) as vscode.CompletionList;
+        const pos = new vscode.Position(13, 18);
+        const list = await retryUntil(
+            async () => {
+                const cmd = 'vscode.executeCompletionItemProvider';
+                const list = await vscode.commands.executeCommand<vscode.CompletionList>(cmd, doc.uri, pos);
+                if (
+                    list &&
+                    list.items.length > 0 &&
+                    list.items.some((item) => item.kind === vscode.CompletionItemKind.Method)
```
Comment (ybiquitous) [inline src/test/suite/extension.test.ts:30]: [note] `list.items` sometimes include only `CompletionItemKind.Text` items. So, this check using `some()` is necessary.

File: src/test/suite/extension.test.ts [hunk 2]
```diff
@@ -95,22 +107,22 @@ suite('go to definitions', () => {
 
     test('go to initialize method', async () => {
         const doc = await openTargetFile(path.join(simpleProgramPath, 'student.rb'));
-        const loc = (await vscode.commands.executeCommand(
+        const loc = await vscode.commands.executeCommand<vscode.Location[]>(
             'vscode.executeDefinitionProvider',
             doc.uri,
             new vscode.Position(10, 16),
-        )) as vscode.Location[];
+        );
```
Comment (ybiquitous) [inline src/test/suite/extension.test.ts:114]: [note] This refactoring seems out of the scope, but I believe it's important to unify the code style. Specifying `<type>` can reduce extra `()` and `as`.

**Comment by ono-max** [comment]
Really appreciate your great work!

**Pull request merged**